MAUI/Calendar/date-navigations.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: Date Navigations in  MAUI Calendar widget | Syncfusion
+description: Learn here all about Date navigations feature of Syncfusion MAUI Calendar (SfCalendar) widget and more.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Date Navigations in MAUI Calendar (SfCalendar)
+
+## Programmatic date navigation
+Allows you to navigate through the Dates by programmatically in the calendar widget by using the [DisplayDate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_DisplayDate) property of [SfCalendar](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html).
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+{% include_relative code-snippet/month-view.xaml %}
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.DisplayDate = DateTime.Now.AddMonths(2).Date;
+
+{% endhighlight %}
+{% endtabs %}
+
+![display-date-in-maui-calendar](images/date-navigation/display-date-in-maui-calendar.png)
+
+## Programmatic view navigation
+Allows you to navigate through the views by programmatically in the calendar widget by using the [View](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_View) property of [SfCalendar](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html).
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+{% include_relative code-snippet/month-view.xaml %}
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.View = CalendarView.Month;
+
+{% endhighlight %}
+{% endtabs %}
+
+![monthview-in-maui-calendar](images/date-navigation/monthview-in-maui-calendar.png)
+
+## Allow view navigation
+Allows you to navigate by using the `AllowViewNavigation` property through tap interaction on the cell or header. By using this property you can able to restrict the view navigation and allow you to select the cells in the Year, Decade and Century views.
```

Comment:
It allows you to navigate using the `AllowViewNavigation` property by tapping interaction on the cell or header. By using this property, you can restrict the view navigation and allow you to select the cells in the Year, Decade, and Century views.

Reply:
Content modified.